Makefile

```diff
@@ -44,7 +45,6 @@ test-tier2:
 
 # TIER3
 test-tier3:
-	TIER3=1 MAVEN_TESTAPP_USER=$(GITHUB_USER) MAVEN_TESTAPP_TOKEN=$(GITHUB_TOKEN) $(MAKE) test-analysis
```

Comment:
Is the TIER3 analysis test going ot be executed in other target or we're getting rid of analysis test on non-public environment?

Reply:
TIER3 is being executed on Jenkins as PR check and there is an open task to run it as nightly as well against the `main` branch

Reply:
Ok, thanks!